src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/Workspace.cs

```diff
@@ -385,17 +391,11 @@ private async Task OnBuildUpdateAsync(IProjectVersionedValue<BuildUpdate> update
 
         return;
 
-        async void ApplyProjectBuild(
+        void ApplyProjectBuild(
```

Comment:
The `async void` in the previous code was an incomplete refactor on my part, fixed here.

We have an analyzer that should have flagged this, but it doesn't work for local functions apparently. I've filed https://github.com/microsoft/vs-threading/issues/1074 to track that.

Reply:
Submitted https://github.com/microsoft/vs-threading/pull/1075 to extend the analyzer to cover local functions and async lambdas.